bruig/flutterui/bruig/lib/screens/newconfig/move_old_version.dart

```diff
@@ -10,9 +10,9 @@ const _errorMoveMsg = "We were unable to move you existing BR wallet/db due "
     "%LOCALAPPDATA%/bruig\n"
     "Please resolve this conflict and then restart Bison Relay";
 
-const _warnMsg1 = "There is an existing, old version of"
-    "the wallet. To use that old version you need to move it"
-    "to a new location.  It is advised to make a backup of"
+const _warnMsg1 = "There is an existing old version of "
```

Comment:
```suggestion
const _warnMsg1 = "There is an existing version of "
```

Maybe just personal preference, but existing old seems redundant and sounds odd to me.

Reply:
kk